utility/script_config_dict.py

```diff
@@ -0,0 +1,430 @@
+# Copyright The IETF Trust 2023, All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__author__ = 'Dmytro Kyrychenko'
+__copyright__ = 'Copyright The IETF Trust 2023, All Rights Reserved'
+__license__ = 'Apache License, Version 2.0'
+__email__ = 'dmytro.kyrychenko@pantheon.tech'
+
+import os
+from datetime import datetime
+
+from utility.create_config import create_config
+from utility.staticVariables import backup_date_format
+
+config = create_config()
+credentials = config.get('Secrets-Section', 'confd-credentials').strip('"').split()
+save_file_dir = config.get('Directory-Section', 'save-file-dir')
+result_dir = config.get('Web-Section', 'result-html-dir')
+
+script_config_dict = {
```

Comment:
Try and give this a type hint, so the type checker can check that all the args are in the correct format. You might need a new `TypedDict` class to make it work. Without checks, editing a blob this big could lead to a lot of mistakes.